### DISCUSSION TITLE
Unable to transfer domain out

### DISCUSSION THREAD

**Discussion by Froidland** [vercel/community#5826] [Help] [answered]
### Summary

Since February 9th, I've been trying to transfer my out domain from Vercel to Porkbun, but have been stuck with the same “bad authcode” message. Is there any way to reset the auth code?

[Previous post](https://github.com/orgs/vercel/discussions/5795)

### Steps to Reproduce

1. Select my domain.
2. Press the transfer out button.
3. Copy the auth code.
4. Paste the code in the registrar's transfer form.
5. Receive “bad authcode” error message after a few seconds.

**Comment by amyegan** [comment]
Hi @Froidland. What is the domain you're trying to transfer?

**Reply by Froidland** [reply]
Hi, that would be "pancho.moe".

**Reply by amyegan** [reply, marked as answer]
Thanks for sharing that. The auth code is reset now. You can get the new code from your Vercel dashboard.

**Reply by Froidland** [reply]
It seems like that worked, thank you very much!
